backend/src/main/java/de/neufische/mydepot/model/PortfolioItem.java

```diff
@@ -12,11 +12,11 @@
 @NoArgsConstructor
 
 public class PortfolioItem {
-    String id;                                      //von DB
-    String displayName;                             //von Frontend/DB
-    String symbol;                                  //von Frontend/DB
-    double quantity;                                //von Frontend/DB
-    double boughtAtPricePerShare;                   //von Frontend/DB
-    BigDecimal regularMarketPrice;                  //aus API
-    BigDecimal regularMarketChangePercent;          //aus API
+    private String id;                                      //von DB
+    private String displayName;                             //von Frontend/DB
+    private String symbol;                                  //von Frontend/DB
+    private double quantity;                                //von Frontend/DB
+    private double boughtAtPricePerShare;                   //von Frontend/DB
+    private BigDecimal regularMarketPrice;                  //aus API
+    private BigDecimal regularMarketChangePercent;          //aus API
 }
```

Comment:
i miss the type of portfolioItem. A portfolio is a collection of  stocks, bonds, commodities or cash. i think when you have only stocks a better name of Class i StockItem?